vue-components/package.json

```diff
@@ -48,12 +48,12 @@
     "@types/jest": "^24.0.19",
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.33.0",
-    "@vue/cli-plugin-babel": "~4.4.0",
+    "@vue/cli-plugin-babel": "~4.5.0",
```

Comment:
What is the motivation behind pinning some of those to their minor version (`~`) and some to their major version (`^`)?

Reply:
The pinning to minor done by default when you run `vue create` (with whatever vue cli version you are using). I assume they do this to avoid peer dependencies falling out of sync, as it seems the plugins and vue cli are all cut from the same source and versioned together.

But now that you mentioned it, I see I accidentally pinned `@vue/cli-plugin-typescript` to major (`^`) instead of minor (`~`), thanks!

Reply:
Good point. But given that point, would it make sense to raise the `@vue/cli-plugin-e2e-nightwatch` dependency to `4.5.x` as well?

Reply:
Great question, and yes, I will change the entry to 4.5 and up. I'm leaving the pin to major (`^`) As it seems to have been introduced by the installation of this package itself: when you run `vue add e2e-nightwatch` (after boiler-plating your project without it) the package.json entry is added as pinned to major (I agree that this is odd, but I assume the vue cli team have their reasons).